examples/basic-tls/tidb-cluster.yaml

```diff
@@ -0,0 +1,61 @@
+# IT IS NOT SUITABLE FOR PRODUCTION USE.
+# This YAML describes a basic TiDB cluster with minimum resource requirements,
+# which should be able to run in any Kubernetes cluster with storage support.
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: basic-tls
+spec:
+  tlsCluster:
+    enabled: true
+  version: v4.0.8
+  timezone: UTC
+  pvReclaimPolicy: Delete
+  enableDynamicConfiguration: true
+  configUpdateStrategy: RollingUpdate
+  discovery: {}
+
+  pd:
+    baseImage: pingcap/pd
+    replicas: 1
+    requests:
+      storage: "1Gi"
+    config:
+      security:
+        cert-allowed-cn:
+          - TiDB
+
+  tikv:
+    baseImage: pingcap/tikv
+    replicas: 1
+    requests:
+      storage: "1Gi"
+    config:
+      security:
+        cert-allowed-cn:
+          - TiDB
+      storage:
+        # In basic examples, we set this to avoid using too much storage.
+        reserve-space: "0MB"
+      rocksdb:
+        # In basic examples, we set this to avoid the following error in some Kubernetes clusters:
+        # "the maximum number of open file descriptors is too small, got 1024, expect greater or equal to 82920"
+        max-open-files: 256
+      raftdb:
+        max-open-files: 256
+
+  tidb:
+    baseImage: pingcap/tidb
+    replicas: 1
+    service:
+      type: NodePort
+    tlsClient:
+      enabled: true
+    config:
+      security:
+        cert-verify-cn:
+          - TiDB
+
+  ticdc:
+    baseImage: pingcap/ticdc
+    replicas: 1
```

Comment:
we do not enable `ticdc` in `basic` and `basic-cn`, should we need to enable it here?

Reply:
This is actually in PR #3594 which is not merged yet 😲 
Why is it here...

Reply:
Deleted in both PR, thanks 😂